client/me/security-checkup/index.jsx

```diff
@@ -48,29 +46,8 @@ import {
 
 import { getCurrentUserEmail } from 'state/current-user/selectors';
 
-const SecurityCheckup = React.createClass( {
-	displayName: 'SecurityCheckup',
-
-	mixins: [ observe( 'userSettings' ) ],
-
-	componentDidMount: function() {
-		this.props.userSettings.getSettings();
-	},
-
-	render: function() {
-		const twoStepEnabled = this.props.userSettings.isTwoStepEnabled();
-
-		const {
-			translate,
-		} = this.props;
-
-		const twoStepNoticeMessage = translate(
-			'To edit your SMS Number, go to {{a}}Two-Step Authentication{{/a}}.', {
-				components: {
-					a: <a href="/me/security/two-step" />
-				},
-			} );
-
+class SecurityCheckup extends Component {
+	render() {
```

Comment:
Ah! It should be. Thanks for pointing that out. Fixed by f4308e9c6a973db5dd012c98c8b2eaa10046fb2d.